src/webgpu/util/color_space_conversion.ts

```diff
@@ -0,0 +1,195 @@
+import { assert } from '../../common/util/util.js';
+
+import { multiplyMatrices } from './math.js';
+
+/**
+ * The color space conversion function definations are fully aign with
+ * CSS Color Module Level 4 Sample Code: https://drafts.csswg.org/css-color/#color-conversion-code
+ */
+
+/**
+ *  Sample code for color conversions
+ * Conversion can also be done using ICC profiles and a Color Management System
+ * For clarity, a library is used for matrix multiplication (multiply-matrices.js)
+ */
+
+// sRGB-related functions
+
+/**
+ * convert an array of sRGB values
+ * where in-gamut values are in the range [0 - 1]
+ * to linear light (un-companded) form.
+ * https://en.wikipedia.org/wiki/SRGB
+ * Extended transfer function:
+ * for negative values,  linear portion is extended on reflection of axis,
+ * then reflected power function is used.
+ */
+function lin_sRGB(RGB: Array<number>) {
```

Comment:
`conversion.ts` already has `gammaCompress` and `gammaDecompress`, though unfortunately they behave somewhat differently (they're for srgb formats, not real color management), so they're not necessarily easy to clean up. We should at least leave a MAINTENANCE_TODO in both places pointing to the other, so we can maybe clean it up later.

Reply:
Ok, the origin plan here is that "we move the code from the CSS spec and that's why it is trusted". So I don't change anything(considering the spec is in draft and might have changes in that part).

Reply:
Oh, sorry, I didn't realize this was copied directly from that spec. In that case, I'm happy to keep all this code exactly as it was in the source.

Reply:
(except still best to have doc comments)

Reply:
Done.